### PR TITLE
Add pagination to pipeline executions

### PR DESCRIPTION
**Pull request by alice485**
- Add table pagination
- Use MUI v4 since v5 requires higher version of React than what Deck is on
- Reorganize components

### CODE REVIEW

File: spin-observatory-plugin-deck/src/components/pipelines/ExecutionsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+} from '@material-ui/core';
+import React, { ChangeEvent, useState } from 'react';
+import { IExecution } from '@spinnaker/core';
+import { TableHeaders } from './TableHeaders';
+import { ExecutionRow } from './ExecutionRow';
+import { IStatus, STATUSES, DEFAULT_ROWS_PER_PAGE } from './constants';
+import { PaginationActions } from './PaginationActions';
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles({
```
Comment (victor-homedepot) [inline spin-observatory-plugin-deck/src/components/pipelines/ExecutionsTable.tsx:19]: You don't like stylesheets? lol

File: spin-observatory-plugin-deck/src/index.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { PluginContainer } from './components/PluginContainer';
 
 export const plugin: IDeckPlugin = {
   initialize: () => {
+    console.log("v0.5.0+28")
```
Comment (victor-homedepot) [inline spin-observatory-plugin-deck/src/index.ts]: probably want to get rid of this

**Review by victor-homedepot — COMMENTED** [review]
LGTM. Just get rid of the log.

File: spin-observatory-plugin-deck/src/components/pipelines/ExecutionsTable.tsx (hunk 1 above)
Comment (alice485) [reply spin-observatory-plugin-deck/src/components/pipelines/ExecutionsTable.tsx:19]: just felt like there were few enough style changes that it made sense to keep them in the same files

**Review by victor-homedepot — APPROVED** [review]

**Pull request merged**